### PR TITLE
Add SBD users to tsp staging

### PR DESCRIPTION
**Pull request by LeDeep**
## Description

Adds a new user to office staging.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

To download the secure migration and view its contents:
```sh
download-secure-migration 20190515164502_add-sbd-tsp-users.sql
cat tmp/secure_migrations/*/20190515164502_add-sbd-tsp-users.sql
```

To check that the migration has run correctly:
```
- make run_staging_migrations
- psql-prod-migrations
- Query database to confirm users have been added
```

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/165896132) for this change

### CODE REVIEW

**Comment by mikena-truss** [comment]
Does something need to be changed (timestamp maybe?) to make sure this runs after the office user change? Maybe it just will because that one has already run and the migration system keeps track of that rather than file names...

**Comment by chrisgilmerproj** [comment]
@LeDeep - in your instructions instead of

```
SECURE_MIGRATION_BUCKET_NAME=transcom-ppp-app-staging-us-west-2 run-prod-migrations
```

You ought to just have people run `make run_staging_migrations`. It reduces the overhead of having to deal with the bucket name. And if that doesn't work then let me know and we'll fix it.

And maybe in another PR we could modify `scripts/upload-secure-migrations` to take an `ENVIRONMENT=staging` variable so that you can use it directly instead of using the `aws s3 cp` command directly.

File: migrations/20190515164502_add-sba-tsp-users-051509.up.fizz [hunk 1]
```diff
@@ -0,0 +1 @@
+exec("./apply-secure-migration.sh 20190515164502_add-sba-tsp-users-051509.sql")
```
Comment (chrisgilmerproj) [inline migrations/20190515164502_add-sba-tsp-users-051509.up.fizz]: Should these files be named `add-sbd` instead of `add-sba`?

**Comment by chrisgilmerproj** [comment]
Also, for anyone who wants to see the migration:

```sh
download-secure-migration 20190515164502_add-sba-tsp-users-051509.sql
cat tmp/secure_migrations/*/20190515164502_add-sba-tsp-users-051509.sql
```

And then I checked the new row additions with:

```sql
select * from tsp_users where email ilike '%@slicedbreaddesign.com';
```

**Comment by LeDeep** [comment]
> Does something need to be changed (timestamp maybe?) to make sure this runs after the office user change? Maybe it just will because that one has already run and the migration system keeps track of that rather than file names...

Since the office migration already ran, I think we will be fine. When I tested the prod migrations locally everything it worked as expected.

**Comment by LeDeep** [comment]
> Also, for anyone who wants to see the migration:
> 
> ```shell
> download-secure-migration 20190515164502_add-sba-tsp-users-051509.sql
> cat tmp/secure_migrations/*/20190515164502_add-sba-tsp-users-051509.sql
> ```
> 
> And then I checked the new row additions with:
> 
> ```sql
> select * from tsp_users where email ilike '%@slicedbreaddesign.com';
> ```

Making the instructions clearer is a good idea. I'll update.

**Comment by chrisgilmerproj** [comment]
> > Does something need to be changed (timestamp maybe?) to make sure this runs after the office user change? Maybe it just will because that one has already run and the migration system keeps track of that rather than file names...
> 
> Since the office migration already ran, I think we will be fine. When I tested the prod migrations locally everything it worked as expected.

Also, they are slightly different:

```
    20190515160313_add-sba-office-user-051509.up.fizz
    20190515164502_add-sba-tsp-users-051509.up.fizz
```

File: migrations/20190515164502_add-sba-tsp-users-051509.up.fizz (hunk 1 above)
Comment (chrisgilmerproj) [reply migrations/20190515164502_add-sba-tsp-users-051509.up.fizz]: seems like the other file was named `sba` so I'm gonna say leave it alone.

**Comment by LeDeep** [comment]
> Actually, can you modify the names of the migrations to remove the date at the end? They already have a date at the beginning of the migration file name.

Since there are quite a few office/tsp migrations, we've been adding the date at the end so all the migrations aren't named exactly the same. I can still remove it though, if you think it's unnecessary.

**Comment by chrisgilmerproj** [comment]
> > Actually, can you modify the names of the migrations to remove the date at the end? They already have a date at the beginning of the migration file name.
> 
> Since there are quite a few office/tsp migrations, we've been adding the date at the end so all the migrations aren't named exactly the same. I can still remove it though, if you think it's unnecessary.

I think it's unnecessary since the date is already at the front of the migration.

**Pull request merged**